test/Airdrop.spec.ts

```diff
@@ -1,15 +1,82 @@
 import { ethers } from "ethers";
 import * as utils from '../scripts/helpers/utils'
+import { setup } from '../scripts/helpers/setup'
 const RollupCore = artifacts.require("Rollup");
+const RollupUtils = artifacts.require("RollupUtils");
+const IMT = artifacts.require("IncrementalTree");
 
 contract("airdrop", async function () {
 
-    it("lets coordinator submit a batch", async function () {
+    const tokenType = 1
+    const epoch = 1
+
+    xit("lets coordinator submit a batch", async function () {
```

Comment:
I hesitate to give testing for even the validity path right now. The iteration of writing tests is a little bit slow.
 
Reasons:
- Every time you rerun a single test, it needs two minutes to redeploy every contract.
- Struct doesn't work specifically in tests but not in go-node or remix. See  https://github.com/thehubbleproject/contracts/issues/57. We have to rewrite the rollup utils format just to let the tests pass.
- It's difficult to create objects like Merkle proof or siblings manually. This suggests we need more utils or domain objects